### PR TITLE
Add chart data

### PR DESCRIPTION
**Pull request by stg101**
- Add collaborators state
- Add hard-coded data

Close #12

### CODE REVIEW

File: src/views/chart.js [hunk 1]
```diff
@@ -2,7 +2,40 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 
+const initialColaborators = JSON.parse(
+  localStorage.getItem("colaborators") ||
+    JSON.stringify([
+      {
```
Comment (test0n3) [inline src/views/chart.js]: Lets set a really good model to test the drawing.

File: src/views/chart.js [hunk 2]
```diff
@@ -2,7 +2,40 @@
...
+        id: 0,
+        name: "Carlos Ayala",
+        position: "CEO",
+        parent: "",
+        children: []
```
Comment (DiegoCuevas) [inline src/views/chart.js]: Why did not you put the names of your children here?

Comment (stg101) [reply src/views/chart.js]: Oh thanks , I forget about them

File: src/views/chart.js (hunk 1 above)
Comment (stg101) [reply src/views/chart.js]: Thanks for the advice, we will do it in the next pull request

**Pull request merged**